### ISSUE TITLE
[BUG] License file missing from wheel

### ISSUE THREAD

**Issue by johnthagen** [cython/cython#3854]
**Describe the bug**

Tools such as [`pip-licenses`](https://github.com/raimon49/pip-licenses) use the LICENSE file included in uploaded Python packages to generate license reports. The latest release of `Cython` does not have a LICENSE file included in some wheels, which makes it more challenging to comply with the license.

Could a new release of `Cython` be released using the latest `wheel` library that pulls in the LICENSE file? Generally, ensuring that `wheel` is updated to the latest before building the wheel is enough to pull in the LICENSE file as recent versions of `wheel` will automatically look in the root project directory for files named LICENSE*, or COPYING*, etc.

You can see the LICENSE missing from the `dist-info` folder of the unzipped wheel:

```
% ls -l
total 8
drwxr-xr-x@ 20 u  g   640B Oct  1 07:36 Cython/
drwxr-xr-x@  9 u  g   288B Oct  1 07:36 Cython-0.29.21.dist-info/
-rw-rw-r--@  1 u  g   520B Jun 22  2015 cython.py
drwxr-xr-x@  5 u  g   160B Oct  1 07:36 pyximport/
% ls -l Cython-0.29.21.dist-info 
total 104
-rw-rw-r--@ 1 u  g   1.1K Jul  8 21:45 DESCRIPTION.rst
-rw-rw-r--@ 1 u  g   2.6K Jul  8 21:45 METADATA
-rw-rw-r--@ 1 u  g    26K Jul  8 21:45 RECORD
-rw-rw-r--@ 1 u  g   110B Jul  8 21:45 WHEEL
-rw-rw-r--@ 1 u  g   140B Jul  8 21:45 entry_points.txt
-rw-rw-r--@ 1 u  g   1.8K Jul  8 21:45 metadata.json
-rw-rw-r--@ 1 u  g    24B Jul  8 21:45 top_level.txt
```

**To Reproduce on Ubuntu 18.04**

```bash
$ sudo apt install python3-venv
$ python3 -m venv venv
$ source venv/bin/activate
(venv) $ pip install -U pip setuptools wheel
(venv) $ pip install pip-licenses Cython
(venv) $ $ pip freeze --all
Cython==0.29.21
pip==20.2.3
pip-licenses==2.3.0
pkg-resources==0.0.0
PTable==0.9.2
setuptools==50.3.0
wheel==0.35.1
(venv) $ pip-licenses --from=mixed --format=plain-vertical --no-license-path --with-license-file
Cython
0.29.21
Apache Software License
UNKNOWN

...
```

**Expected behavior**

It is expected the LICENSE file is included in every wheel and source distribution for Cython. The `UNKNOWN` in the `pip-licenses` command would then be replaced with the contents of `LICENSE.txt`.

Note: I was able to verify the LICENSE file is included in the Mac wheel, so there just may be a CI setting that needs to get the Linux wheels up to the latest wheel before building.

**Environment (please complete the following information):**
 - OS: Ubuntu 18.04
 - Python version: 3.6.9
 - Cython version: 0.29.21

**Comment by scoder** [comment]
> which makes it more challenging to comply with the license.

Given that, statistically, no-one on earth reads licenses, allow me to express a scent of doubt in this regard. :)

Also note how your `pip-license` command actually prints the right license name, just not its (easy-to-find-elsewhere) content.

That said, if there is something we can improve, I'm happy to look at a PR.

**Comment by johnthagen** [comment]
> Also note how your pip-license command actually prints the right license name, just not its (easy-to-find-elsewhere) content.

Yes, `pip-licenses` finds the license name identified from the trove classifiers. These are properly specified in `setup.py`:

https://github.com/cython/cython/blob/af757997c7d13c00b579fd00393350d9367dd12a/setup.py#L271

> Given that, statistically, no-one on earth reads licenses, allow me to express a scent of doubt in this regard. :)

I completely agree. It's more for those who want to be sure they are actually abiding by the license when they bundle `Cython` with something they distribute, since even permissive licenses like Apache, require that you retain the license itself when you redistribute.

**Comment by johnthagen** [comment]
I can confirm this has been fixed in 0.29.22. 🎉
